drafts/concise_guide.ipynb

```diff
@@ -92,7 +92,66 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When creating a response, FastHTML will automatically render tags using the to_xml function. "
+    "This example demonstrates key aspects of how FastHTML components handle attributes:\n",
+    "\n",
+    "```python\n",
+    "Label(\n",
+    "    \"Choose an option\", \n",
+    "    Select(\n",
+    "        Option(\"one\", value=\"1\", selected=True),  # Boolean True renders just the attribute name\n",
+    "        Option(\"two\", value=\"2\", selected=False), # Boolean False omits the attribute entirely\n",
+    "        Option(\"three\", value=3),                 # Non-string values are converted to strings\n",
+    "        cls=\"selector\",                           # 'cls' becomes 'class' in HTML (Python reserved word)\n",
+    "        _id=\"counter\",                            # Leading underscore is stripped during rendering\n",
+    "        **{'@click':\"alert('Clicked');\"},         # Dict unpacking for attributes with special characters\n",
+    "    ),\n",
+    "    _for=\"counter\",                               # '_for' becomes 'for' (can also use 'fr')\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "FastHTML components can be linked by matching `_id` with `_for` attributes. The library handles Python reserved words by using alternatives like `cls` for `class` and stripping leading underscores during rendering."
```

Comment:
`id` and `for` matching is html behavior so doesn't need to be mentioned. The attr aliases were already mentioned earlier -- you could add the extra details to that part instead.